draft-ietf-httpbis-message-signatures.md

```diff
@@ -991,13 +1022,19 @@ Signature: sig1=:P0wLUszWQjoi54udOtydf9IWTfNhy+r53jGFj9XZuP4uKwxyJo\
   BNFv3r5S9IXf2fYJK+eyW4AiGVMvMcOg==:
 ~~~
 
+It is RECOMMENDED that the `Signature` HTTP field be included as a header, though it is possible to include it as a trailer to facilitate signing a message after its content has been processed by the signer.
```

Comment:
I'd recall - if it's not done elsewhere - that intermediaries can legitimately remove trailers

Reply:
Added language to both places talking about trailers.